care/facility/api/serializers/daily_round.py

```diff
@@ -94,6 +94,24 @@ class Meta:
         )
         exclude = ("deleted",)
 
+    def validate_bp(self, value):
+        if not value:
+            return value
+
+        systolic = value.get("systolic")
+        diastolic = value.get("diastolic")
+        mean = value.get("mean")
```

Comment:
this could be an issue, the log updates filed by the middleware does not contain the mean value

what would be the default in case of normal log updates?

Reply:
MAP = 1/3 × SBP + 2/3 × DBP

Reply:
@sainak Updated the schema to disallow `mean` from being stored to DB, and simplified validations too.